config/initializers/datetime_formats.rb

```diff
@@ -1,5 +1,5 @@
 Date::DATE_FORMATS.merge!(
-  date: ->(d) {
+  participant_date: ->(d) {
     d.strftime("%m/%d/%Y")
```

Comment:
@michaelwehrley I'm probably missing something but `participant_date` should be `d.strftime("%b %d %Y)"`